py/torch_migraphx/fx/tracer/acc_tracer/acc_ops.py

```diff
@@ -1984,6 +1984,62 @@ def isinf(*, input):
     return torch.isinf(input=input)
 
 
+@register_acc_op
+def any(*, input, dim=None, keepdim=False):
+    if dim is not None:
+        return torch.any(input, dim=dim, keepdim=keepdim)
+    return input.any(dtype=dtype)
+
+
+@register_custom_acc_mapper_fn(
+    op_and_target=("call_method", "any"),
+    arg_replacement_tuples=[
+        ("input", "input"),
+        ("dim", "dim", this_arg_is_optional),
+        ("keepdim", "keepdim", this_arg_is_optional),
+    ],
+)
+@register_custom_acc_mapper_fn(
+    op_and_target=("call_function", torch.any),
+    arg_replacement_tuples=[
+        ("input", "input"),
+        ("dim", "dim", this_arg_is_optional),
+        ("keepdim", "keepdim", this_arg_is_optional),
+    ],
+)
+def any_mapper(node: torch.fx.Node,
+               mod: torch.fx.GraphModule) -> torch.fx.Node:
+    return reduce_op_mapper(node, mod, any)
+
+
+@register_acc_op
+def all(*, input, dim=None, keepdim=False):
+    if dim is not None:
+        return torch.all(input, dim=dim, keepdim=keepdim)
+    return input.all(dtype=dtype)
```

Comment:
```suggestion
    return input.all()
```
all doesnt take dtype param